ITEM.tsv

```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
 ITEM_20150714_005513	A strong potion used for removing odors. It has an antiseptic scent to it. 
 ITEM_20150714_005514	Tilla Monastery Scriptures 
 ITEM_20150714_005515	Scriptures that were scattered by rampaging monsters. Bring them to Mohaim. 
-ITEM_20150717_005516	
-ITEM_20150717_005517	
+ITEM_20150717_005516	The strong blade seems to show the way of the Sword.
```

Comment:
Btw, how do I edit the pull request? Or is the admin. responsible for doing that?